pkg/ocm/storage/received/ocm.go

```diff
@@ -414,3 +445,26 @@ func (d *driver) CreateStorageSpace(ctx context.Context, req *provider.CreateSto
 func (d *driver) UpdateStorageSpace(ctx context.Context, req *provider.UpdateStorageSpaceRequest) (*provider.UpdateStorageSpaceResponse, error) {
 	return nil, errtypes.NotSupported("operation not supported")
 }
+
+// Cleanup function to remove expired cache entries
+func (d *driver) cleanupCache() {
```

Comment:
Why not use `ttlcache.Cache` here? Then you don't have to manually do cache cleanups like this, and we already use it in several places in Reva

Reply:
Good point, will do!